sdk/python/feast/embedded_go/online_features_service.py

```diff
@@ -0,0 +1,120 @@
+from typing import Any, Dict, List, Optional, Union
+
+import pyarrow as pa
+from pyarrow.cffi import ffi
+
+from feast.errors import FeatureNameCollisionError
+from feast.feature_service import FeatureService
+from feast.online_response import OnlineResponse
+from feast.protos.feast.serving.ServingService_pb2 import GetOnlineFeaturesResponse
+from feast.protos.feast.types.Value_pb2 import RepeatedValue, Value
+from feast.repo_config import RepoConfig
+from tests.unit.test_proto_json import FeatureVector
```

Comment:
I think this import should be from `feast.protos.feast.serving.ServingService_pb2.GetOnlineFeaturesResponse.FeatureVector`, not from the tests

Reply:
Great catch. Fixed